package.json

```diff
@@ -28,5 +29,10 @@
     "babel-preset-stage-0": "^6.24.1",
     "rimraf": "^2.6.2",
     "semistandard": "^12.0.1"
+  },
+  "dependencies": {
+    "@mapbox/mapbox-gl-draw": "^1.1.1",
```

Comment:
They are already peerDependencies. Add these 3 as devDependencies for TS to resolve paths.

Reply:
For some reason, I feel like I had to `npm install @mapbox/mapbo-gl-draw --save` because `npm install` didn't install them. Maybe it's because I didn't use yarn. Maybe it was Friday and I wasn't thinking clearly. :) I'll swap them to devdependencies, though.

Reply:
Just FYI, in my next commit, I left the typings as a regular dependency, per http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies. You can change that if you want.